src/Common/src/System/Net/CookieParser.cs

```diff
@@ -614,7 +614,8 @@ internal Cookie Get()
                                     {
                                         domainSet = true;
                                         cookie.Domain = CheckQuoted(_tokenizer.Value);
-                                        cookie.IsQuotedDomain = _tokenizer.Quoted;
+                                        // TODO: this is internal.
+                                        // cookie.IsQuotedDomain = _tokenizer.Quoted;
```

Comment:
We don't have a way to solve this problem yet, I guess we can go with reflection option, add to this file [here](https://github.com/dotnet/corefx/blob/master/src/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs), and then explore options to expose the APIs tracked in issue #13607

cc @stephentoub @davidsh